core/Tracker/VisitExcluded.php

```diff
@@ -125,12 +126,17 @@ public function isExcluded()
         }
 
         // Check if Referrer URL is a known spam
-        if (!$excluded) {
-            $excluded = $this->isReferrerSpamExcluded();
-            if ($excluded) {
-                Common::printDebug("Referrer URL is blacklisted as spam.");
+        $generalConfig = Config::getInstance()->General;
+        if (!$generalConfig['disable_spam_excluding'])
```

Comment:
@john357smith could we name the setting `enable_spam_filter` and put this into the `Tracker` section? We're trying to have most of our settings start with `enable` rather than `disable` .

Reply:
Parameter changed to enable_spam_filter.